examples_tests/39.DenoiserTonemapper/CommandLineHandler.cpp

```diff
@@ -192,8 +211,8 @@ CommandLineHandler::CommandLineHandler(core::vector<std::string> argv, IAssetMan
 						if (variablesHandle.size() != TA_COUNT)
 							variablesHandle.resize(TA_COUNT);
 
-						reference.second[TA_KEY_VALUE] = variablesHandle[TA_KEY_VALUE].empty() ? std::string("0") : variablesHandle[TA_KEY_VALUE];
-						reference.second[TA_EXTRA_PARAMETER] = variablesHandle[TA_EXTRA_PARAMETER].empty() ? std::string("0") : variablesHandle[TA_EXTRA_PARAMETER];
+						reference.second[TA_KEY_VALUE] = variablesHandle[TA_KEY_VALUE].empty() ? std::string("0.4") : variablesHandle[TA_KEY_VALUE];
```

Comment:
tonemapper parameters are required, don't even bother with defaults... two parameters and tonemapping method must always be present

Reply:
it will be going through validation, done